src/module/api/update/search/index.ts

```diff
@@ -0,0 +1,31 @@
+import * as apigents from "@venturemark/apigents";
+import * as env from "module/env";
+import * as spec from "./spec";
+
+export function Search(req: spec.Req): spec.Res {
```

Comment:
This is the implementation of the Search function. I had problems getting the typing right and I bet it still is not correct. Need to try to run this and then we need to pair a little on that. The input/output structure is defined in our `apischema`. See https://github.com/venturemark/apischema/blob/main/pbf/update/search.proto.